docs/config_search.md

```diff
@@ -18,84 +18,132 @@ limitations under the License.
-- [Quick](config_search.md#quick-search-mode) is the default for ensemble models or when profiling multiple models concurrenty, and will use heuristics to try to find the optimal configuration much quicker than brute, and can be enabled via `--run-config-search-mode quick`
-- [Multi-model](config_search.md#multi-model-search-mode) will profile mutliple models to find the optimal configurations for all models while they are running concurrently. This feature is enabled via `--run-config-profile-models-concurrently-enable`
+- [Brute Force Search](config_search.md#brute-search-mode)
+  - `Search type:` Brute-force sweep of the cross product of all possible configurations
+  - `Default for:`
+    - Single non-ensemble models
+    - Multiple models being profiled sequentially
+  - `Command:` `--run-config-search-mode brute`
+- [Quick Search](config_search.md#quick-search-mode)
+  - `Search type:` Heuristic sweep using a hill-climbing algorithm to find an optimal configuration
+  - `Default for:`
+    - Single ensemble models
+    - Multiple models beging profiled concurrently
+  - `Command:` `--run-config-search-mode quick`
+- [Concurrent / Multi-model Search](config_search.md#multi-model-search-mode)
+  - `Search type:` Heuristic concurrent sweep of all models' search spaces to find an optimal configuration for all models
+  - `Default for: `
+    - None
+  - `Command:` `--run-config-profile-models-concurrently-enable`
 
 ## Brute Search Mode
 
-Model Analyzer's brute search mode will do a brute-force sweep of the cross product of all possible configurations. You can [Manually](config_search.md#manual-brute-search) provide `model_config_parameters` to tell Model Analyzer what to sweep over, or you can
-let it [Automatically](config_search.md#automatic-brute-search) sweep through configurations expected to have the highest impact on performance for Triton models.
+---
 
-This search mode is the default when profiling non-ensemble models sequentially.
+**Default search mode when profiling non-ensemble models sequentially**
 
-### Automatic Brute Search
+Model Analyzer's brute search mode will do a brute-force sweep of the cross product of all possible configurations. <br>
+It has two modes:
 
-Automatic configuration search is the default behavior when running Model
-Analyzer without manually specifying what values to search. The parameters
-that are automatically searched are
-[`max_batch_size`](https://github.com/triton-inference-server/server/blob/master/docs/user_guide/model_configuration.md#maximum-batch-size)
-and
-[`instance_group`](https://github.com/triton-inference-server/server/blob/master/docs/user_guide/model_configuration.md#instance-groups).
-Additionally, [`dynamic_batching`](https://github.com/triton-inference-server/server/blob/master/docs/user_guide/model_configuration.md#dynamic-batcher) will be enabled if it is legal to do so.
+- [Automatic](config_search.md#automatic-brute-search)
+  - **No** model config parameters are specified
```

Comment:
Yes, I believe these statements are accurate. Note there is a difference btw. `model config parameters` (controls instance/batch size) and `parameters` (controls concurrency) which is definitely a source of confusion.

Reply:
ok, manual vs automatic is just referring to model config parameters? That makes sense to me, because we aren't really "searching" for an optimal concurrency.

Reply:
Wait, i'm still confused. In the Manual Brute Search section it says if any 'parameters' are specified that you must specify batch sizes/instance counts/concurrencies you want to sweep. Does that mean that specifying parameters puts us in manual mode?  Does the user have to specify those options in `model config parameters`?